### PR TITLE
Created multiple blocks and adjusted the display of user names and build button.

### PR DESCRIPTION
**Pull request by RaaviSindhu**
Created different blocks and adjusted the display of players names to fit the screen.

### CODE REVIEW

**Review by late-in-autumn — APPROVED** [review]
Seems to work. But I suggest holding this change back for now. We need to manually resolve merge conflicts anyways, so I suggest merging this after #27 has merged.

**Review by JavierMHdz — APPROVED** [review]

**Review by JavierMHdz — APPROVED** [review]

**Review by JavierMHdz — APPROVED** [review]

**Pull request merged**
